src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+class App extends Component{
+
+    constructor(props){
+        super(props);
+
+        this.state = {};
+    }
+    render(){
+        return(
+            <div>
+           <h1>I am App and I render display and ButttonPanel</h1>
+           <Display />
+           <ButtonPanel/>
+            </div>
```

Comment:
code not well formatted....check linting again

Reply:
I tried to fix that locally but the linter still fails in the respository